src/frontend/tsx/Chessboard.tsx

```diff
@@ -139,6 +138,7 @@ export function Chessboard () {
         }
 
         //  XX: uh-oh
+        //      new type guard here
         if ((displayGame as ActiveGameInfo).gotDrawOffer) {
```

Comment:
using `(displayGame as ActiveGameInfo)` became necessary in a few locations where `ArchivedGameInfo` wouldn't fit.